src/java/main/org/apache/zookeeper/server/persistence/FileTxnLog.java

```diff
@@ -255,30 +255,19 @@ private void padFile(FileOutputStream out) throws IOException {
      */
     public static File[] getLogFiles(File[] logDirList,long snapshotZxid) {
         List<File> files = Util.sortDataDir(logDirList, "log", true);
```

Comment:
couldn't we make the last param false and then we can use a simpler `for (File f : files)` loop?

Reply:
Agree, I'll make the change, thanks.